wandb/proto/wandb_internal.proto

```diff
@@ -886,6 +886,7 @@ message JobInfoResponse {
 message LogArtifactRequest {
   ArtifactRecord artifact = 1;
   int64 history_step = 2;
+  string staging_dir = 3;
```

Comment:
we will probably need to have some logic versioning sdk client and nexus, since your current code will break with earlier version of sdk.